WalletWasabi.Documentation/WasabiCompatibility.md

```diff
@@ -22,6 +22,7 @@ This document lists all the officially supported software and devices by Wasabi
 - Ledger Nano X
 - Trezor Model T
 - Blockstream Jade
+- BitBox02
```

Comment:
Not directly related to this PR, but as there are more and more signing devices supported, wouldn't it make sense to sort this list alphabetically?

Reply:
thats right. okey i will

Reply:
Write the only one that we are surely supporting.